ItemLibrary/Smartphone.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ItemLibrary
+{
+    public class Smartphone : Item
+    {
+        public string FrontCamera { get; set; }
+        public string EndCamera { get; set; }
+        public double ScreenDiagonal { get; set; }
+        public int Storage { get; set; }
+        public int RAM { get; set; }
+        public string Processor { get; set; }
+
+        public Smartphone ()
+        {
+        }
+
+        //will remove this method in near future
```

Comment:
I think you can delete this already

Reply:
will not delete it yet, because these methods are used in other classes, that is why don't want to lose working functionality